test/BaselineOutput/Common/EntryPoints/core_manifest.json

```diff
@@ -21710,6 +21710,103 @@
         "ITransformOutput"
       ]
     },
+    {
```

Comment:
Can we add a test actually invoking the entry points for each supported ml task?